tests/login.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/login');
+  await page.getByLabel('Email').fill('test@gmail.com');
+  await page.getByLabel('Password').fill('testtest');
+});
+
+test.describe('Login', () => {
+  test('should redirect to homepage', async ({ page }) => {
+    await page.getByRole('button', { name: 'Log in' }).click();
+    await page.waitForURL('/');
+    expect(page.url()).toBe('http://localhost:5173/');
```

Comment:
Credo che dovresti far in modo di usare url relative (tipo:  '/' per la Home, '/user' per la pagina user etc.)
[Playwright ha una sua utility per controllare la url](https://playwright.dev/docs/api/class-pageassertions#page-assertions-to-have-url)

Reply:
Allora Gabri purtroppo con:   await expect(page).goto('/'); mi da problemi a trovare l'url. Ci avevo già provato, adesso provo a cercare altra documentazione su playwright

Reply:
`await expect(page).goto('/')` non mi torna, forse intendevi `await page.goto('/')`?